src/Fixer/ArrayNotation/NoTrailingCommaInSinglelineArrayFixer.php

```diff
@@ -84,7 +84,7 @@ private function fixArray(Tokens $tokens, $index)
 
         if ($beforeEndToken->equals(',')) {
             $tokens->removeTrailingWhitespace($beforeEndIndex);
-            $beforeEndToken->clear();
```

Comment:
it doesn't matter for performance
and we often do things like here - have both `$fooIndex` and `$fooToken` vars